src/unity/python/turicreate/toolkits/object_detector/object_detector.py

```diff
@@ -96,8 +96,8 @@ def _raise_error_if_not_detection_sframe(dataset, feature, annotations, require_
 
 
 def create(dataset, annotations=None, feature=None, model='darknet-yolo',
-           classes=None, batch_size=0, max_iterations=0, verbose=True,
-           **kwargs):
+           grid_shape=[13, 13], classes=None, batch_size=0, 
```

Comment:
To avoid breaking existing API calls that rely on argument ordering, please add `grid_shape` last after `verbose` (and before `**kwargs`). Python allows explicit kwargs to be passed either by keyword or argument ordering, so existing calls may be assuming that the 5th argument is `classes` (and so on).